tests/opentracer/test_tracer_asyncio.py

```diff
@@ -92,3 +98,85 @@ def coro():
         eq_(1, len(traces[0]))
         eq_('coroutine', traces[0][0].name)
 
+
+
+class TestTracerAsyncioCompatibility(AsyncioTestCase):
```

Comment:
Correct that you used what we have, but maybe this one is worth checking to make the testing easier: https://github.com/pytest-dev/pytest-asyncio

If it works fine, I think you can have plain functions with fixtures only.

Reply:
Unfortunately `pytest-asyncio` does not support versions of python < 3.5 as of version 0.6.0: https://github.com/pytest-dev/pytest-asyncio#060-2017-05-28 😐else it would have been great to use.